src/OpenDiffix.Core.Tests/Expression.Tests.fs

```diff
@@ -535,8 +535,15 @@ let colRef2 = ColumnReference(2, RealType)
 
 let evaluate expr = Expression.evaluate testRow expr
 
+let aggContext =
+  {
+    AnonymizationParams = AnonymizationParams.Default
+    GroupingLabels = [||]
+    Aggregators = [||]
+  }
+
 let evaluateAggregator aggSpec args =
-  evaluateAggregator (ExecutionContext.makeDefault ()) aggSpec args testRows
+  evaluateAggregator (aggContext, Some { BucketSeed = 0UL }) aggSpec args testRows
```

Comment:
Potentially dumb question, but can we make the `Some { BucketSeed = 0UL }` part of `aggContext`? Any specific reason to pass them around separately?

Reply:
The two components of this context have different scopes: the aggregation context applies to the current aggregation operation and the anonymization context applies to the current bucket. It seems natural to keep them separate. In the case of evaluating aggregators, we could create another context structure to contain both components, but that adds another level of indirection and pollutes the type namespace. I wasn't convinced it is worth doing until now.

Reply:
The "AnonymizationContext" type used here should be replaced with a "BucketContext" type, to make it more clear it applies to each individual bucket. The two are identical for now (they might not be in the future), so I postponed adding another type to reduce type pollution. But maybe that wasn't the best decision...

Reply:
We already have a ton of "context"s around so the simpler the better!